src/components/contextual/pages/vebal/LMVoting/LMVoting.vue

```diff
@@ -195,10 +187,42 @@ watch(
         </h3>
       </div>
     </div>
-    <ResubmitVotesAlert
+
+    <VotingAlert v-if="veBalLockTooShort" title="veBAL not locked for 7 days">
```

Comment:
There's a bunch of new alerts that we were showing in the old voting modal. Now they are shown above the voting list to explain why the user cannot click "Vote".